interfaces/HO-Portal/src/assets/i18n/en.json

```diff
@@ -287,6 +287,10 @@
         "unused-vouchers": {
           "btn-text": "Export list of unused vouchers",
           "confirm-message": "This will download a CSV-file with all unused voucher(s).<br /><br />Note that this action can take a while to complete. Do not leave this page in the meantime."
+        },
+        "duplicate-phone-numbers": {
+          "btn-text": "Export list of duplicate phone-numbers",
+          "confirm-message": "This will download a CSV-file with all PA that share any phone-number."
```

Comment:
all PAs?

Reply:
And maybe add a short explanation like 'You can subsequently filter on these PAs and reject them"?

Reply:
PA = People Affected is already plural right?

Reply:
Hmm you are technically maybe right, but in my interpretation PA stands for singular, and thus PAs for plural. Also, because there is otherwise no difference, which is confusing. 

On another note, you could actually argue we should minimize the short-forms completely, so we should say People Affected here then?